.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: Build CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: configure
+      run: ./configure
```

Comment:
Why? There is no need for a config step.

Reply:
because of this:
https://github.com/pheiduck/TLP/actions/runs/3982116677/jobs/6826299107#step:3:4